src/main/java/de/tum/in/www1/artemis/web/rest/QuizSubmissionResource.java

```diff
@@ -153,7 +153,7 @@ public QuizSubmissionResource(QuizExerciseService quizExerciseService, QuizSubmi
         quizSubmission.calculateAndUpdateScores(quizExercise);
 
         // create Participation stub
-        Participation participation = new Participation().exercise(quizExercise);
+        StudentParticipation participation = new StudentParticipation().exercise(quizExercise);
```

Comment:
I think this is an example that shows that we can be specific in some places about the Participations: This can never a Template/SolutionParticipation, so it's better to be clear here :+1: .